### PR TITLE
modified script so it should work even with fb bot

### PR DESCRIPTION
**Pull request by WesleyYep**
same commands  
`sh lex.sh deploy` to deploy existing bot, now will work even if alias or fb link is present
`sh lex.sh save` saves bot to config files in repo

### CODE REVIEW

File: lex.sh [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
-## SAVE ##
-if [ "$1" = "save" ]; then # save or deploy
-    echo saving lex
-    aws lex-models get-bot --name MovieBot --version-or-alias "\$LATEST" > lex/MovieBot.json
-    # get intents
-    node > lex/intents/intents.txt <<EOF
-var data = require('./lex/MovieBot.json');
+LEXPATH='lex'
+if [ "$1" = "deploy" ]; then #save to hidden .lex folder to use for deployment checksums
+    cp -r lex .lex
```
Comment (changkon) [inline lex.sh]: I put the tests in lex folder. maybe need to filter it?

Comment (WesleyYep) [reply lex.sh]: oh yeah ok

**Comment by WesleyYep** [comment]
Oh hold on, forgot to test for new intents/slots

**Review by changkon — APPROVED** [review]

**Comment by changkon** [comment]
If the script gets too complex, how about like a python script. I know it gets tricky on linux cause of the EOF. idk why

**Pull request merged**